contessa/normalizer.py

```
@@ -44,15 +45,21 @@ def normalize(cls, rules_def):
     def _should_normalize(rule_def):
         if "columns" in rule_def:
             return True
-        elif "time_filters" in rule_def and len(rule_def["time_filters"]) > 1:
+        elif "separate_time_filters" in rule_def:
+            if len(rule_def["separate_time_filters"]) <= 1:
+                logging.error(
```

lets raise the Error here